crates/wysiwyg/src/composer_model/format.rs

```diff
@@ -190,6 +267,98 @@ where
         }
     }
 
+    fn unformat_several_nodes(
+        &mut self,
+        start: usize,
+        end: usize,
+        range: &MultipleNodesRange,
+        format: InlineFormatType,
+    ) {
+        for location in range.locations.iter() {
+            let node = self.state.dom.lookup_node(&location.node_handle);
+            if let DomNode::Container(node) = node {
+                match node.kind() {
+                    ContainerNodeKind::Formatting(f) => {
+                        if f.clone() == format.clone() {
```

Comment:
This seems to work fine:

```suggestion
                        if *f == format {
```